library/extractor/src/main/java/com/google/android/exoplayer2/text/ssa/SsaStyle.java

```diff
@@ -227,6 +245,15 @@ public static Integer parseColor(String ssaColorExpression) {
     return Color.argb(a, r, g, b);
   }
 
+  public static float parseMargin(String floatValue) {
+    try {
+      return Float.parseFloat(floatValue);
+    } catch (NumberFormatException e) {
+      Log.w(TAG, "Failed to parse margin value: '" + floatValue + "'", e);
+      return 0f;
```

Comment:
I think `DIMEN_UNSET` would be more consistent here, no?

Reply:
Probably it would be more consistent, however in case of margins, the default value is `0` as I understand. Also setting `DIMEN_UNSET` would lead to a bunch of extra checks before applying the margins. In this way we can just add/subtract the margins without any checks, as `0` won't have an effect. But I can go with `DIMEN_UNSET` if you say so. :)

Reply:
Yep, I agree we never need to use `DIMEN_UNSET` because `0` implies 'fall back to the next level of defaults' - thanks for clarifying!